### DISCUSSION TITLE
How to config local search when using `defineConfigWithTheme` ?

### DISCUSSION THREAD

**Discussion by gantrol** [vuejs/vitepress#3639] [Q&A]
I download the [vuejs docs repo](https://github.com/vuejs/docs), and want to change the search from algolia to local, but it seems no `search` attribute in `themeConfig` of `defineConfigWithTheme`

![image](https://github.com/vuejs/vitepress/assets/31330732/b5caa993-9673-4782-8cad-b0f61ab2a808)

I have search it in the issues, docs, google..., but no solution.

**Comment by brc-dd** [comment]
https://github.com/vuejs/docs/discussions/2724
